README.md

```diff
@@ -512,7 +512,8 @@ Note that all heuristics could be applied if and only if same preprocessing was
 {"models": [
     {"path": "<first_word2vec_file>", "binary": true, "trainable": false,
```

Comment:
Looks like `binary` property is not supported now.

Please fix here and in other examples.